src/blog/the-truffle-nft-box-all-you-need-to-know/index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: The Truffle NFT Box. All you need to know
+hide:
+  - navigation
+---
+
+**By [Kingsley Arinze](https://www.linkedin.com/in/bruzzy)**
+
+![The Truffle NFT Box. All you need to know](./nft-box.jpg)
+
+Infura [announced](https://blog.infura.io/post/introducing-the-infura-nft-api-beta-release) the private beta release of their NFT API a few months ago. Since then, there has been an ecosystem of tools developing around it, making it easier to build with Infura’s NFT API. 
+
+One such tool is the [Truffle NFT Box](https://trufflesuite.com/boxes/nft-box), which as you’ll see later in this post, utilizes another tool built on top of the Infura NFT API, the Infura NFT SDK, to make NFT development as stress-free as possible.
+
+In this post, we will discuss the Infura NFT API, as well as the Infura NFT SDK which is available for download as an [NPM package](https://www.npmjs.com/package/@infura/sdk). We’ll then explore the Truffle NFT Box and how it makes NFT development a lot easier by providing the necessary configuration and boilerplate code needed to start and finish an NFT project.
+
+## Infura NFT API, what is it?
+
+In the past year, we’ve seen a significant increase in cryptocurrency adoption across the world, which is in part, a result of the NFT boom of the past year. 
+
+As a matter of fact, there is an estimated 880% increase in global crypto adoption in 2021, a $180 million market capitalization for stablecoins as of March 2022 as well as a $40 million market capitalization for NFTs in 2021.
+
+We’ve also seen a number of popular brands like Coca-Cola, Adidas, Nike, and Formula 1 to name a few, launch NFTs and invest in the Metaverse. 
+
+All this points to the fact that NFTs are an important part of our ecosystem and their use cases aren’t just limited to PFPS or artwork, but extend to include digital identity verification, membership/access to gated digital and physical communities, and much more.
+
+![use cases and consumer trends beyond "owning jpegs"](./ownership.png)
+
+Infura and Truffle understand the importance of NFTs and where it is headed, which is why they’ve expanded their product offering to include the [NFT API](https://docs.api.infura.io/nft), a set of READ RESTful APIs that enable developers to: 
```

Comment:
Switch "why they’ve expanded their product offering" to "we've expanded our product offering"?

Also the "READ" in "READ RESTful APIs" should probably be removed given it also supports "writes".